csharp/src/Ice/IObjectPrx.cs

```diff
@@ -446,6 +428,55 @@ protected static async Task IceReadVoidResponseAsync(Task<IncomingResponseFrame>
         protected static async Task<T> IceReadResponseAsync<T>(Task<IncomingResponseFrame> task,
                                                                InputStreamReader<T> reader)
             => (await task.ConfigureAwait(false)).ReadReturnValue(reader);
+
+        private static OutgoingRequestWithNonStructParam<string, bool>? _iceIsA_RequestValue;
+        private static OutgoingRequestWithNonStructParam<string, bool> _iceIsA_Request
+        {
+            get
+            {
+                if (_iceIsA_RequestValue == null)
```

Comment:
These fields/property should use the same naming convention as the generated code.